docs/5g-on-helium/5g-on-helium.mdx

```diff
@@ -80,6 +80,8 @@ As defined in [HIP-51](https://github.com/helium/HIP/blob/main/0051-helium-dao.m
 the Helium Foundation receives an allocation of 50B (billion) of 250B MOBILE tokens in the form of a
 pre-mine.
 
+[HIP-75](https://github.com/helium/HIP/blob/main/0075-mobile-poc-initiate-programmatic-minting-and-updated-emissions-curve.md#economic-changes-to-emissions-curve) further modifiled the emission schedule and started the programatic minting of MOBILE.
```

Comment:
Would be good to just lay out the emissions schedule rather than pointing to HIPs

Reply:
added emissions table in change [0ca2a35](https://github.com/helium/docs/pull/1159/commits/0ca2a358e345cd585a2e088860c67edfefd01900)